src/test/resources/org/jenkinsci/plugins/xunit/types/phpunit/testcase11/input.xml

```diff
@@ -0,0 +1,73 @@
+    <testcase file="/var/www/insly/tests/acceptance/Signup/signupBaseCest.php" name="baseTestInstanceCreation" class="Signup\signupBaseCest" feature="base test instance creation" assertions="0" time="0.110998">
+      <error type="Facebook\WebDriver\Exception\UnknownServerException">signupBaseCest: Base test instance creation
+Facebook\WebDriver\Exception\UnknownServerException: unknown error: unhandled inspector error: {"code":-32000,"message":"Cannot navigate to invalid URL"}
+  (Session info: chrome=68.0.3440.84)
+  (Driver info: chromedriver=2.41.578700 (2f1ed5f9343c13f73144538f15c00b370eda6706),platform=Linux 4.14.91+ x86_64)
+
+/var/www/insly/vendor/facebook/webdriver/lib/Exception/WebDriverException.php:114
+/var/www/insly/vendor/facebook/webdriver/lib/Remote/HttpCommandExecutor.php:326
+/var/www/insly/vendor/facebook/webdriver/lib/Remote/RemoteWebDriver.php:547
+/var/www/insly/vendor/facebook/webdriver/lib/Remote/RemoteWebDriver.php:226
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Module/WebDriver.php:891
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Step.php:265
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Scenario.php:72
+/var/www/insly/tests/_support/_generated/AcceptanceTesterActions.php:283
+/var/www/insly/tests/_support/Page/Signup/MainPage.php:19
+/var/www/insly/tests/acceptance/Signup/signupBaseCest.php:25
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Lib/Di.php:127
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Test/Cest.php:138
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Test/Cest.php:150
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Test/Cest.php:82
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Test/Test.php:89
+/var/www/insly/vendor/codeception/phpunit-wrapper/src/Runner.php:110
+/var/www/insly/vendor/codeception/codeception/src/Codeception/SuiteManager.php:158
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Codecept.php:192
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Codecept.php:181
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Command/Run.php:495
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Command/Run.php:390
+/var/www/insly/vendor/symfony/console/Command/Command.php:255
+/var/www/insly/vendor/symfony/console/Application.php:901
+/var/www/insly/vendor/symfony/console/Application.php:262
+/var/www/insly/vendor/symfony/console/Application.php:145
+/var/www/insly/vendor/codeception/codeception/src/Codeception/Application.php:108
+</error>
+    </testcase>
+    <testcase name="testWarning" class="PHPUnit\SelfTest\Basic\StatusTest" classname="PHPUnit.SelfTest.Basic.StatusTest" file="/var/test/StatusTest.php" line="42" assertions="0" time="42.001">
+      <warning type="PHPUnit\Framework\Warning">PHPUnit\SelfTest\Basic\StatusTest::testWarning
```

Comment:
you can trasform warning messages in system-err element like:
```
<system-err>PHPUnit\SelfTest\Basic\StatusTest::testWarning

%s%eStatusTest.php:%d</system-err>
```

Reply:
This looks like a good alternative (assuming I'm understanding it correctly). Thanks.

PR updated.